stdlib/enum.pyi

```diff
@@ -2,24 +2,86 @@ import sys
 import types
 from abc import ABCMeta
 from builtins import property as _builtins_property
-from typing import Any, Iterator, Type, TypeVar
+from typing import Any, Dict, Iterable, Iterator, Mapping, Tuple, Type, TypeVar, Union, overload
```

Comment:
Nit: `Iterable`, `Iterator`, and `Mapping` should be imported from `collections.abc`.

Reply:
Done! 🙂